### DISCUSSION TITLE
Water Sprinkler Setup

### DISCUSSION THREAD

**Discussion by darwiethepang02** [firemodels/fds#13641] [Q&A]
I would like to ask how can I setup my water sprinkler using pressure and also just to double confirm, for the code below, the diameter for the liquid particle is in terms of microns right? Also, is the pressure unit in Pa?

PROPERTIES OF WATER SPRINKLER
&SPEC ID = 'WATER VAPOR' /
&PART ID = 'my droplets', DIAMETER = 50., SPEC_ID = 'WATER VAPOR' /
&PROP ID = 'K-11', QUANTITY = 'SPRINKLER LINK TEMPERATURE', RTI = 148., C_FACTOR = 0.7,
ACTIVATION_TEMPERATURE = 68., OFFSET = 0.10, PART_ID = 'my droplets', OPERATING_PRESSURE = 8000000.,
ORIFICE_DIAMETER = 0.020, SPRAY_ANGLE = 0.,60., SMOKEVIEW_ID = 'sprinkler_upright' /

**Comment by mcgratta** [comment]
There are tables in the back of the User's Guide with a list of parameters and units.

**Comment by Myrmecio** [comment]
Check the units. The operating pressure is entered in bars, and 8 million bars is several times the pressure at the center of the Earth (approx. 3.6 million bars). I hope this helps.
